### PR TITLE
Create LIT6822SalamLakaAqabi.xml

### PR DESCRIPTION
**Pull request by CarstenHoffmannMarburg**
I created a new ID of LIT6822SalamLakaAqabi, attested in BMLacq776 and discussed in issue [#2345](https://github.com/BetaMasaheft/Documentation/issues/2345).

### CODE REVIEW

File: new/LIT6822SalamLakaAqabi.xml [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="CH" when="2023-05-10">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <body>
```
Comment (eu-genia) [inline new/LIT6822SalamLakaAqabi.xml:54]: please remove the second `body`, your file is invalid

**Review by eu-genia — CHANGES_REQUESTED** [review]

File: new/LIT6822SalamLakaAqabi.xml [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+                <div type="edition" xml:lang="gez">
+                        <note>The following text is taken from <ref type="mss" corresp="BMLacq776#ms_i1.2.3"/>.</note>
+                        <ab>
+                            ሰላም፡ ለካ፡ አቃቢ፡ ቅዱሳን፡ እምተሐውኮ፡ ወለፀረ፡ ነፍሶሙ፡ ዘሞቃሕኮ፡ አመ፡ መጽአ፡ ወልድ፡
```
Comment (thea-m) [inline new/LIT6822SalamLakaAqabi.xml:58]: ለከ

File: new/LIT6822SalamLakaAqabi.xml [hunk 3]
```diff
@@ -0,0 +1,72 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6822SalamLakaAqabi" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Salām la-ka ʿaqābi</title>
```
Comment (thea-m) [inline new/LIT6822SalamLakaAqabi.xml:8]: laka 
could you update the title to have fidal and transliteration?

**Review by thea-m — CHANGES_REQUESTED** [review]
Thank you!

File: new/LIT6822SalamLakaAqabi.xml (hunk 2 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6822SalamLakaAqabi.xml:58]: Done.

File: new/LIT6822SalamLakaAqabi.xml (hunk 3 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6822SalamLakaAqabi.xml:8]: I deleted the hyphen, so now it is "laka" (not "la-ka"). 
How can I give the title in fidal in addition to the title in transliteration? Most of the Salam, that I checked have an English title without transliteration or fidal title.

Comment (thea-m) [reply new/LIT6822SalamLakaAqabi.xml:8]:
```
  <title xml:lang="gez" xml:id="t1">ሰላም፡ ለከ፡ ዐቃቢ፡</title>
  <title xml:lang="gez" corresp="#t1">Salām laka ʿaqābi...</title>
```

(Adding ... to titles that are simply the first words of the text in question isn't prescribed anywhere, but Denis taught it to me for the DS Catalogue, and I find it very reasonable)

Comment (CarstenHoffmannMarburg) [reply new/LIT6822SalamLakaAqabi.xml:8]: Me too, I find it very reasonable and did it in the way, you recommended!

File: new/LIT6822SalamLakaAqabi.xml (hunk 1 above)
Comment (CarstenHoffmannMarburg) [reply new/LIT6822SalamLakaAqabi.xml:54]: Done.

**Review by eu-genia — APPROVED** [review]

**Pull request merged**